story.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
oof, I just noticed that the link is to `/story` but this file is `story.html`, so the link won't work. I like the URL, so can you please move this file into a new 'story' folder and rename it  to `index.html`, so that it sits at `/story/index.html`? That way the URL should work perfectly. Thanks!